src/main/java/frc/robot/subsystems/module/Module.java

```diff
@@ -96,7 +98,7 @@ private void setSpeed(SwerveModuleState desiredState, boolean isOpenLoop) {
             double velocity = ConversionUtils.falconToRPM(ConversionUtils.MPSToFalcon(desiredState.speedMetersPerSecond, DriveConstants.kWheelCircumference,
                 DriveConstants.kDriveGearRatio), 1)/60;
             // TODO: This curently doesn't use the feedforward.
-            driveMotor.setControl(new VelocityDutyCycle(velocity).withEnableFOC(true));
+            driveMotor.setControl(new VelocityVoltage(velocity).withEnableFOC(true).withFeedForward(feedforward.calculate(velocity)));
```

Comment:
Why do you define a VelocityVoltage as a variable and then create a new one every frame? Either use the variable or delete it.

Reply:
I changed it to using the existing variable.